src/screens/Books/components/ListItem/index.js

```diff
@@ -31,7 +32,8 @@ ListItem.propTypes = {
     author: PropTypes.string,
     imageLink: PropTypes.string,
     link: PropTypes.string
-  })
+  }),
+  onPress: PropTypes.func
```

Comment:
I would add `.isRequired`:

```js
ListiItem.propTypes = {
  /* Other PropTypes */
  onPress: PropTypes.func.isRequired
};
```

This allows you to check if there's any case where the `onPress` function was `undefined`.